CHANGELOG.md

```diff
@@ -23,6 +23,9 @@
 - Enforce minimum length of 1 character for the jailer ID.
 - Exit with error code when starting the jailer process fails.
 
+### Removed
+- Removed `InstanceHalt` from the list of possible actions.
```

Comment:
nit: mdl reports `(stdin):27: MD032 Lists should be surrounded by blank lines` meaning blank line after **Removed** header

Reply:
Done.